react-app/src/hooks/useFetch.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as I from '../interfaces';
+
+const useFetch = (apiCall: Function): [Array<I.Restaurant>, boolean, string] => {
```

Comment:
I like the idea of a wrapper of API calls. I'm looking forward to reviewing this on Tuesday.